tests/wpt/web-platform-tests/xhr/abort-after-send.any.js

```diff
@@ -9,7 +9,7 @@
         client.addEventListener("readystatechange", test.step_func(function() {
           if(client.readyState == 4) {
             control_flag = true
-            assert_equals(client.responseXML, null)
+            assert_equals(test.GLOBAL.isWindow(), false)
```

Comment:
Not quite! The test should still check client.responseXML in the case when isWindow() is true, but in the case when it's false we should not bother checking anything.

Reply:
To be clear, the `.any.js` suffix on this test file means that the same test code will be executed in a window environment and in a worker environment, so it needs to be a valid test in both environments.

Reply:
Ahh sorry, I misinterpreted what you meant. Just to be clear you are talking about something like this:
```
          if(client.readyState == 4 && test.GLOBAL.isWindow()) {
            control_flag = true
            assert_equals(client.responseXML, null)
            assert_equals(client.responseText, "")
            assert_equals(client.status, 0)
            assert_equals(client.statusText, "")
            assert_equals(client.getAllResponseHeaders(), "")
            assert_equals(client.getResponseHeader('Content-Type'), null)
          }
```
Or are you talking about only checking client.responsXML individually like so:
```
          if(client.readyState == 4) {
            control_flag = true
            if (test.GLOBAL.isWindow()) {
              assert_equals(client.responseXML, null)
            }
            assert_equals(client.responseText, "")
            assert_equals(client.status, 0)
            assert_equals(client.statusText, "")
            assert_equals(client.getAllResponseHeaders(), "")
            assert_equals(client.getResponseHeader('Content-Type'), null)
          }
```

Reply:
I was talking about only checking responseXML individually, like your second sample.